src/transformers/models/deta/modeling_deta.py

```diff
@@ -23,6 +23,8 @@
 
 import torch
 import torch.nn.functional as F
+from accelerate import PartialState
+from accelerate.utils import reduce
```

Comment:
This needs to be behind a protected import:

```
if is_accelerate_available():
    from accelerate ... 
```

Reply:
modified!